docs/Python-for-Formula-Authors.md

```diff
@@ -40,7 +43,7 @@ If you submit a formula with this syntax to core, you may be asked to rewrite it
 
 ## Applications
 
-`ansible.rb` and `jrnl.rb` are good examples of applications that follow this advice.
+[`ansible.rb`][Formula/ansible.rb] and [`jrnl.rb`][Formula/jrnl.rb] are good examples of applications that follow this advice.
```

Comment:
I think you can either remove this section or remove the links to application above.